.github/CODEOWNERS

```diff
@@ -26,7 +26,7 @@
 /src/Grpc/**/PublicAPI.*Shipped.txt                                               @dotnet/aspnet-api-review @JamesNK @captainsafia @mgravell
 /src/Hosting/                                                                     @tratcher @halter73
 /src/Hosting/**/PublicAPI.*Shipped.txt                                            @dotnet/aspnet-api-review @tratcher
-/src/Http/                                                                        @tratcher @BrennanConroy @halter73
+/src/Http/                                                                        @tratcher @BrennanConroy @halter73 @captainsafia
```

Comment:
Does it make sense to put you under a subfolder, so you only get pinged for RequestDelegate stuff?